BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/security/rest/mappers/PermissionMapper.java

```diff
@@ -23,6 +24,11 @@
 public class PermissionMapper {
 
     private static final String STATUS_OK = "OK";
+    private final MagicFolderUtil magicFolderUtil;
+
+    public PermissionMapper(MagicFolderUtil magicFolderUtil) {
```

Comment:
Since this class is marked as a component, is it necessary to create a new constructor?